src/notation/inotationinteraction.h

```diff
@@ -117,6 +117,13 @@ class INotationInteraction
 
     virtual void addText(TextType type) = 0;
     virtual void addFiguredBass() = 0;
+
+    virtual void addStretch(qreal value) = 0;
+
+    virtual void resetStretch() = 0;
+    virtual void resetTextStyleOverrides() = 0;
```

Comment:
Maybe better to unite those methods in one?

void resetToDefault(ResettableValueType type)

enum ResettableValueType {
Stretch,
BeamMode,
ShapesAndPosition,
TextStyleOverriders
}

Reply:
thanks, fixed